examples/compose-bird/build.gradle.kts

```diff
@@ -0,0 +1,28 @@
+// Add compose gradle plugin
+plugins {
+    kotlin("multiplatform") version "1.5.10"
+    id("org.jetbrains.compose") version "0.0.0-web-dev-14"
```

Comment:
Could you please update the version to 0.5.0-build225? :) 

You'll also need to update the ui code since that build brought some breaking changes:
https://github.com/JetBrains/compose-jb/releases/tag/0.5.0-build225-web

Reply:
Sure. resolved in https://github.com/JetBrains/compose-jb/pull/797/commits/4c24f4df09649ebea26214246faa127e7374046a